sh/downgrade-db-2-7-to-2-6.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -euo pipefail
+
+CLI_TOOL=kubectl
+#CLI_TOOL=oc
+NAMESPACE=kube-system
```

Comment:
The default namespace should be `storageos` since that's the default that the plugin uses.

Reply:
I'm not sure I agree, I don't know why this daemonset would need to run in the same ns that the cluster is in

Especially as the instructions to use the script specify uninstalling v2.7.0 before using it so it is somewhat likely that the `storageos` ns won't even exist at this stage.

Reply:
Ah yes good point, I agree with you then, `kube-system` makes more sense.